foundations/installations/installing_node.md

```diff
@@ -8,8 +8,6 @@ Node is also very easy to install using nvm, so this should go quickly :)
 
 ### Installing nvm
 
-### Installation on Linux
```

Comment:
Were these changes requested? I’m not sure why you’ve eliminated some of this content without being prompted.

Reply:
The issue is open and it's not only specific to zsh shell. It's a shell error which may occur in bash also. For that case instead of defining separate steps for nvm installation in mac OS, I merged those steps in handling shell error i.e `nvm command not found` in **Troubleshooting** section.

Reply:
That issue actually should've been closed by a prior PR, apologies.

Reply:
If you like I can do changes, otherwise it's good to close the issue with previous changes.